hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/fees/calculation/AccessorBasedUsagesTest.java

```diff
@@ -283,7 +284,8 @@ void worksAsExpectedForTokenMint() {
 
         subject.assess(sigUsage, txnAccessor, accumulator);
 
-        verify(tokenOpsUsage).tokenMintUsage(sigUsage, baseMeta, tokenMintMeta, accumulator);
+        verify(tokenOpsUsage)
+                .tokenMintUsage(eq(sigUsage), eq(baseMeta), eq(tokenMintMeta), eq(accumulator), any(SubType.class));
```

Comment:
Nit:
```suggestion
                .tokenMintUsage(eq(sigUsage), eq(baseMeta), eq(tokenMintMeta), eq(accumulator), notNull(SubType.class));
```

Looking at how `SubType` is used, seems like `notNull(...)` makes more sense

Reply:
yes, will do in next PR . Thanks @mhess-swl